src/go/rpk/pkg/cli/cmd/container/common/common.go

```diff
@@ -442,3 +453,63 @@ This can happen for a couple of reasons:
 	}
 	return err
 }
+
+func NodeAddr(port uint) string {
+	return fmt.Sprintf(
+		"127.0.0.1:%d",
+		port,
+	)
+}
+
+func RenderClusterInfoStatus(c Client) error {
+	nodes, err := GetExistingNodes(c)
+	if err != nil {
+		return err
+	}
+	if len(nodes) == 0 {
+		log.Info("No Redpanda nodes detected - start some with `rpk container start` or check `docker ps` if you expected there to be some")
+		return nil
+	}
+
+	t := ui.NewRpkTable(log.StandardLogger().Out)
+	t.SetColWidth(80)
+	t.SetAutoWrapText(true)
+	t.SetHeader([]string{"Node ID", "Status", "Kafka Address", "RPC Address", "Admin Address"})
```

Comment:
This looks like the main difference vs. the current output from `start`; I'd add this to `start` and remove the duplicated function here in common.go

Reply:
dropping rpc-address, users should never need to care about this.